### ISSUE TITLE
V7_BCODE_DUMP broken

### ISSUE THREAD

**Issue by joeedh** [cesanta/v7#553]
Hi.  V7_BCODE_DUMP is broken.  The (recent?) addition of immediate strings to the bytecode format has made opcodes that use bcode_decode_lit (OP_PUSH_LIT, OP_SET_VAR, OP_GET_VAR, OP_SAFE_GET_VAR) ambiguous.  This broke dump_op (which does not use bcode_decode_lit, and seems to just assume integer operands).

Thanks.

**Comment by dimonomid** [comment]
Yeah, sorry about that. We actually plan to implement it not as a compile-time option, but as a run-time flag. Hopefully will get it sorted in a few weeks.

**Comment by goniz** [comment]
Any news on this issue?

**Comment by dimonomid** [comment]
Sorry but not yet. We still struggle with other things which considered higher priority.
